### ISSUE TITLE
Impossibilité d'ajouter des plans - suppression de matériels inclus dans des packs

### ISSUE THREAD

**Issue by moutew** [RobertManager/robert#46]
Si les packs matériels contiennent des références de matériels qui ont étés supprimés, une erreur javascript se déclenche dans la fonction displayTekosMatos.

L'erreur se repère après avoir cliqué sur "Ajouter un plan" dans le calendrier. Le curseur de la souris reste une icône de chargement et ne revient pas au pointeur normal. 

Si on affiche le contenu du Pack matériel, les références sont affichées "undefined".

**Comment by polosson** [comment]
Oui, c'est difficile à débugger tout ça, mais pas impossible. J'ajoute qu'à l'étape 4 de l'ajout de plan (le récap), dans la section "matériel", se trouve une catégorie " " (chaîne vide).
